app/views/layouts/application.html.erb

```diff
@@ -111,7 +111,7 @@
             <div class="TutorialToggleHolder"></div>
           <% end %>
       <% end %>
-      <% if @current_user && @current_user.pseudonym.unique_id.downcase != 'teastudent'  %>
+      <% if @current_user && @current_user&.pseudonym&.unique_id&.downcase != 'teastudent'  %>
```

Comment:
What's `teastudent`?

Reply:
according to John and acronym for some school in texas